unit-tests-jest/numbers.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Returns a new array that contains only the even numbers in the given array.
+ */
+export function evenNumbers(numbers: number[]): number[] {
+  return numbers.filter((n) => n % 2 === 0);
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ */
+export function toDollars(amount: number): string {
+  return `$${amount}.00`;
```

Comment:
There's a bug here. It doesn't do cents correctly. You want to use `Number.toFixed()` to fix it.